data/human/free worlds 0 prologue.txt

```diff
@@ -2227,6 +2227,13 @@ mission "Defend Sabik"
 			label paint
 			`	You take the paintbrush, climb on top of your ship, and begin painting it in the green and gold of the Free Worlds. Suddenly you hear a cheer from one of the other ships parked nearby; someone has seen what you are doing and is shouting, "For freedom! Another ship for freedom!" Soon several other people see you, and before long it seems like everyone in the port has picked up the cheer.`
 			`	Tomek stands by, silently, until you finish, and then approaches you with an outstretched arm holding onto a license. "Here. You're going to need this. I'm sure your assistance will be invaluable to our cause," he says as you accept the militia license. "Now if you'll excuse me, I must gather the militia for the coming battle."`
+			choice
+				`	(Let him go.)`
+					goto end
+				`	"I've never worked with another fleet in battle, what advice can you offer?"`
+			`	He stops his turn to leave. "Of course, you probably haven't spent much time in a militia fleet defending a planet or system." After a few moments of thought, he continues, "When fighting alongside a large fleet, it's usually best to work in groups, fire on the same targets as everyone else. You'll see the ships around you working together to disable a single target at a time, instead of everyone trying to run their own one on one fights. We don't expect you to defeat every Navy ship yourself. And, try not to get too close to their ships, either; you probably don't want a group all choosing to target you." He pauses for a moment. "Though, if you're feeling brave, you could try to get a large group of ships to follow you and lead them on a wild goose chase, leaving a smaller force for everyone else to deal with before moving on to your group. It'll be a short chase, however, if you can't stay out of their weapon range and evade their missiles. If it comes to defending this system in particular, the Navy may send additional ships after the initial wave. They will most likely arrive from Kornephoros, so being in their line of fire, on that side of the system, before they arrive is also something to avoid.`
```

Comment:
```suggestion
			`	He stops his turn to leave. "Of course, you probably haven't spent much time in a militia fleet defending a planet or system." After a few moments of thought, he continues, "When fighting alongside a large fleet, it's usually best to work in groups, fire on the same targets as everyone else. You'll see the ships around you working together to disable a single target at a time, instead of everyone trying to run their own one-on-one fights. We don't expect you to defeat every Navy ship yourself. And, try not to get too close to their ships, either; you probably don't want a group all choosing to target you." He pauses for a moment. "Though, if you're feeling brave, you could try to get a large group of ships to follow you and lead them on a wild goose chase, leaving a smaller force for everyone else to deal with before moving on to your group. It'll be a short chase, however, if you can't stay out of their weapon range and evade their missiles. If it comes to defending this system in particular, the Navy may send additional ships after the initial wave. They will most likely arrive from Kornephoros, so being in their line of fire, on that side of the system, before they arrive is also something to avoid.`
```